docs/guide/writing/support-team.rst

```diff
@@ -30,3 +37,34 @@ My advice: don’t be discouraged or alarmed if you get questions you can’t an
 No review process: as a doc writer, the chances are that your work gets edited and proofread by a colleague before publishing. However, when answering most support requests, your input won’t be checked by a colleague.
 
 So, how can you deal with this safety net being removed? Firstly, recognize that it’s fine not to know something – never give an answer you’re not 100% sure about just for the sake of saving face. If you don’t know, it’s always a good idea to ask another agent for their input, or escalate the ticket to the next support level for support.
+
+Tips for tech writers who want to work in their own role within support
+-----------------------------------------------------------------------
+
+If you work for a large-scale SaaS business, being a tech writer involves keeping up with gaps in the documentation, maintaining versions, coordinating release notes, and so on.
+
+Often this means that you have a huge backlog of tasks to juggle, and you cannot take on another role, like supporting client. However, this does not mean that you should lose your contact with this essential part of the business. Working as a technical writer within a support team is a great way to keep up with customer issues while maintain the documentation. All you need is a solid plan to make it happen.
+
+Consider these tips: 
+
+* Get the team on board
+* Make communication simple
+* Collaborate
+
+Get the team on board
+~~~~~~~~~~~~~~~~~~~~~
+
+It is always important to remember that when you work as a separate role within a team, it does not in fact make you apart from the team. Even though you write, and they support, you are still one team who are essential to each other. The better the documentation is, the less redundancies there are in tickets and calls coming in. This means that support will get less tickets, yes, but those tickets will contain challenging issues that really require their time and skills. This is a rewarding, symbiotic relationship. When they have meetings on issues, you attend; when you have doc issues, you communicate those with them. You can’t be a team unless you understand each other and the unique issues each of you face. 
```

Comment:
* Remove extreme words such as "always" and "never" unless they actually mean that.
* Change "as a separate role" to "in a separate role" because you are a working as a person rather than a role.
* Remove "in fact" because the meaning still stands without it, and it comes off too strong with it as is.
* Rephrase "make you apart from" to "separate you from".
* Change "and they support" to "and they provide support" based on a grammar rule; a transitive verb needs an object.
* To avoid difference between American and British English, rephrase "still one team who are" to "still make up one team and are" (In American English, this grammar would be "is" instead of "are"; in British English, it’s "are" instead of "is".
* Change "less" to "fewer" because tickets are countable.
* Rephrase "meetings on issues, you attend;" to "meeting about issues, attend them."
* Rephrase "when you have doc issues, you communicate those to them." to "When you have doc issues, communicate about them with support."
* Replace negative: "You **can’t** be a team unless you understand each other and the unique issues each of you face." with "You can thrive as a team when you understand each other and the unique issues that each of you faces."

Reply:
@osg Thank you very much for all your comments! They are helping me a lot because I'm not an author of this text and I'm not confident in my English.

I have a question about first sentence. Now it looks like this:

> It is important to remember that when you work in a **separate** role within a team, it does not **separate** you from the team.

There are two “separate” in one sentence. Is it all right?

Reply:
Yes, that is fine.